velox/functions/prestosql/tests/SequenceTest.cpp

```diff
@@ -117,3 +117,37 @@ TEST_F(SequenceTest, invalidStep) {
       {startVector, stopVector, stepVector},
       expected);
 }
+
+TEST_F(SequenceTest, dateArguments) {
+  const auto startVector =
+      makeFlatVector<Date>({Date(1991), Date(1992), Date(1992)});
```

Comment:
Can we also test start > stop and start == stop?

Reply:
I think we already have that, we have pairs:
<1991, 1998>
<1992, 1988>
<1992, 1992>